lib/src/canvas/transformations/reference_point.dart

```diff
@@ -0,0 +1,116 @@
+import 'dart:ui';
+/// 
+/// Point, relative to some position inside a container.
+sealed class ReferencePoint {
+  ///
+  /// Returns relative offset depending on [size] of container.
+  Offset value(Size size);
```

Comment:
`value` constructor do provides `Offset`, but all other variants - do not

Reply:
`value` is not a constructor, it is a function that each of the classes should implement. Moved it below all of the constructors to not confuse someone if a future.